sdk/python/pulumi_aws/rds/cluster.py

```diff
@@ -1095,7 +1091,6 @@ def __init__(__self__, *,
         :param pulumi.Input[str] preferred_maintenance_window: Weekly time range during which system maintenance can occur, in (UTC) e.g., `wed:04:00-wed:04:30`
         :param pulumi.Input[str] reader_endpoint: Read-only endpoint for the Aurora cluster, automatically
                load-balanced across replicas
-        :param pulumi.Input[str] replication_source_identifier: ARN of a source DB cluster or DB instance if this DB cluster is to be created as a Read Replica. If DB Cluster is part of a Global Cluster, use the `lifecycle` configuration block `ignore_changes` argument to prevent this provider from showing differences for this argument instead of configuring this value.
```

Comment:
Curious why the docstring is lost here?

Reply:
It's because `replication_source_identifier` needs a replacement, they changed the docstring to include the word `Terraform`. Good thing is the unit tests catch this now after some changes earlier this year :)